server/src/main/java/com/epam/aidial/core/server/controller/ResourceController.java

```diff
@@ -55,6 +65,8 @@ public ResourceController(Proxy proxy, ProxyContext context, boolean metadata) {
         this.accessService = proxy.getAccessService();
         this.lockService = proxy.getLockService();
         this.invitationService = proxy.getInvitationService();
+        this.resourceService = proxy.getResourceService();
```

Comment:
You have already a resource service field: service.

Reply:
removed duplicate